data/issues.py

```diff
@@ -0,0 +1,88 @@
+import logging
+
+import requests
+
+from data.models import (
+    Issue,
+    Label,
+    )
+from data.newcomers import get_newcomers
+from data.models import Contributor
+from data.web import web_url
+
+
+def fetch_issues(hoster):
+    """
+    Get issues opened by newcomers
+
+    :param hoster: a string representing hoster, e.g. 'github'
+    :return: a json of issues data
+    """
+    logger = logging.getLogger(__name__)
+    if hoster == 'github':
```

Comment:
maybe convert to lower case and then compare?

Reply:
Agreed ^^